malaria-ios/SocialShareAbleTest.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  SocialShareAbleTest.swift
+//  malaria-ios
+//
+//  Created by Eric Huang on 1/7/16.
+//  Copyright © 2016 Bruno Henriques. All rights reserved.
+//
+
+import XCTest
+@testable import malaria_ios
```

Comment:
Cool, I didn't have these in my time :D Everything is public as you notice to be testable